app/controllers/providers/dwp/fallback_controller.rb

```diff
@@ -1,20 +1,19 @@
 module Providers
   module DWP
-    class OverridesController < ProviderBaseController
+    class FallbackController < ProviderBaseController
       include ApplicantDetailsCheckable
 
       prefix_step_with :dwp
 
-      helper_method :display_hmrc_text?
-
       def show
-        @form = Providers::DWP::OverridesForm.new(model: partner)
+        # QUESTION: Do we need to delete back hisotry here as the old ConfirmDWPNonPassportedApplicationsController did?
```

Comment:
Have we tested what happens if we click the back button here?

Can this comment be deleted?